src/main/java/org/tarantool/schema/TarantoolSchemaException.java

```diff
@@ -0,0 +1,15 @@
+package org.tarantool.schema;
+
+public class TarantoolSchemaException extends RuntimeException {
```

Comment:
As I see most of our exceptions are runtime. I decided to use the runtime exceptions here because they are more compatible to be used with lambdas. And request arguments use internal resolvers as lambda functions to look up schema ids using string names lazily.

Reply:
Thanks, got it. I'm okay.